documentation/Restart-File.md

```diff
@@ -0,0 +1,64 @@
+# Experimental Feature: Restart File
+
+When using Trapperkeeper apps inside of packages, it is convenient for a service
+framework to have a clear indication as to when all of the Trapperkeeper
+services in the app have been started -- as opposed to just knowing when the
+Java process hosting the app has been spawned.  The "restart file" feature in
+Trapperkeeper provides this capability.  As a reference example, the
+[EZBake](https://github.com/puppetlabs/ezbake) build system for
+Trapperkeeper-based applications makes use of the "restart file" feature.  Its
```

Comment:
This is a total nitpick: I feel like just saying "look at EZBake" is a little mean, since EZBake is pretty complex. Would it be possible to link to one file or to a commit that adds support for this rather than just EZBake as a whole?